Source/Core/EllipsoidGeometry.js

```diff
@@ -206,6 +206,10 @@ define([
         var cosTheta = new Array(slicePartitions);
         var sinTheta = new Array(slicePartitions);
 
+        if ((radii.x === 0) || (radii.y === 0) || (radii.z === 0)) {
```

Comment:
`Ellipsoid.js` does not allow any radii to be less than zero. Should I still make this change?